log.md

```diff
@@ -1,13 +1,13 @@
 # 100 Days Of Code - Log
 
-### Day 0: February 30, 2016 (Example 1)
-##### (delete me or comment me out)
+### Day 1: April 14, 2019 
 
-**Today's Progress**: Fixed CSS, worked on canvas functionality for the app.
 
-**Thoughts:** I really struggled with CSS, but, overall, I feel like I am slowly getting better at it. Canvas is still new for me, but I managed to figure out some basic functionality.
+**Today's Progress**: Learned how to make a simple informational console application informing the user about some characteristics of Python. 
+
+**Thoughts:** Digging a little deeper into Python helps me to understand why it is the world's favored language. Today I learned about the {} .format() method for injecting variables into a string, several ways to use the input() function, using ctrl+e to comment out sections in Geany, and using an if statement with the colon which is a little different than C but still very intuitive. I really think that Python could become my language of choice if I keep diving in.  
 
-**Link to work:** [Calculator App](http://www.example.com)
+**Link to work:** [https://github.com/equitymarkets/100-days-of-code/blob/master/1_Characteristics]
 
```

Comment:
**Today's Progress**: Learned how to use .append() function, import directive, and negative index numbers in a list

Reply:
Thoughts: I am enjoying Python so far. Although small pieces of code seem to take a long time because I need to look things up, once I do it once or twice it is very intuitive.

Reply:
Link to work: [(https://github.com/equitymarkets/100-days-of-code/blob/master/2_Python.py)]